gui/src/components/tracker/TrackerSettings.tsx

```diff
@@ -10,22 +10,22 @@ import {
   ImuType,
   RpcMessage,
 } from 'solarxr-protocol';
-import { useDebouncedEffect } from '../../hooks/timeout';
-import { useTrackerFromId } from '../../hooks/tracker';
-import { useWebsocketAPI } from '../../hooks/websocket-api';
+import { useDebouncedEffect } from '@/hooks/timeout';
+import { useTrackerFromId } from '@/hooks/tracker';
+import { useWebsocketAPI } from '@/hooks/websocket-api';
 import {
   getYawInDegrees,
   MountingOrientationDegreesToQuatT,
-} from '../../maths/quaternion';
-import { ArrowLink } from '../commons/ArrowLink';
-import { BodyPartIcon } from '../commons/BodyPartIcon';
-import { Button } from '../commons/Button';
-import { CheckBox } from '../commons/Checkbox';
-import { WarningIcon } from '../commons/icon/WarningIcon';
-import { Input } from '../commons/Input';
-import { Typography } from '../commons/Typography';
-import { MountingSelectionMenu } from '../onboarding/pages/mounting/MountingSelectionMenu';
-import { IMUVisualizerWidget } from '../widgets/IMUVisualizerWidget';
+} from '@/maths/quaternion';
+import { ArrowLink } from '@/components/commons/ArrowLink';
+import { BodyPartIcon } from '@/components/commons/BodyPartIcon';
+import { Button } from '@/components/commons/Button';
+import { CheckBox } from '@/components/commons/Checkbox';
+import { WarningIcon } from '@/components/commons/icon/WarningIcon';
+import { Input } from '@/components/commons/Input';
+import { Typography } from '@/components/commons/Typography';
+import { MountingSelectionMenu } from '@/components/onboarding/pages/mounting/MountingSelectionMenu';
+import { IMUVisualizerWidget } from '@/components/widgets/IMUVisualizerWidget';
 import { SingleTrackerBodyAssignmentMenu } from './SingleTrackerBodyAssignmentMenu';
```

Comment:
so if its next to it then we dont do path aliasing? :sweat:

Reply:
i dont have a formed opinion on this yet, you can close this one

Reply:
IMO, we should only use `@` if it actually makes the path shorter.

Reply:
the current setting is relative import depth = 1, which means we allow relative import if the file share the same place as the import file